boto/connection.py

```diff
@@ -1112,6 +1115,7 @@ def make_request(self, action, params=None, path='/', verb='GET'):
             http_request.params['Action'] = action
         if self.APIVersion:
             http_request.params['Version'] = self.APIVersion
+        statsd.increment('boto.request', tags=["action:%s" % action or 'unknown', "host:"])
```

Comment:
It would be useful to distinguish the api calls done by our crawlers from api calls done by other hosts we have when they use boto. Do you think it's worth the overhead?